### ISSUE TITLE
"simultaneously used " validation does not have enough information.

### ISSUE THREAD

**Issue by Berserker66** [KhronosGroup/Vulkan-LoaderAndValidationLayers#2093]
In my case I have something like this:

https://pastebin.com/Z7UHMFny

Which unfortunately only tells me that something is wrong but does not actually help me in finding this issue. I logged the thread idents and slowly figured out that it must be my main thread and a thread that syncs the chromium embedded framework texture data. I'm still not sure what is actually happening and I'll need to invest some more time digging.

However, back to this issue - would it be possible to give a bit more information in these threading errors? Such as which command pool and which operations are happening at this time? Spec says, recording, freeing, resetting [...] - telling me which of these it is would already help.

**Comment by mark-lunarg** [comment]
Yeah, it shouldn't be hard to plumb the extra info through, and it's a great idea.  Thanks!
